lib/impure/db_postgres.nim

```diff
@@ -485,6 +487,26 @@ proc insertID*(db: DbConn, query: SqlQuery,
   result = tryInsertID(db, query, args)
   if result < 0: dbError(db)
 
+proc tryInsert*(db: DbConn, query: SqlQuery,pkName: string,
+                  args: varargs[string, `$`]): int64 {.
+                  tags: [WriteDbEffect], since:(1, 3).}=
```

Comment:
```suggestion
proc tryInsert*(db: DbConn, query: SqlQuery,pkName: string,
                args: varargs[string, `$`]): int64
               {.tags: [WriteDbEffect], since: (1, 3).}=
```